### PR TITLE
:arrow_up::one::one: pos_product_available

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_product_available/static/src/xml/pos.xml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--  Copyright 2014-2015 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html). -->
```
Comment (yelizariev) [inline pos_product_available/static/src/xml/pos.xml:3]: Can License be right under Copyright?

**Review by yelizariev — APPROVED** [review]

**Review by ilmir-k — APPROVED** [review]

**Comment by yelizariev** [comment]
Ok, I found the reason: Opening New session takes more than 10 sec sometimes (it's 9sec in example below). Please update timeout on step ``'.o_main_content:has(.loader:hidden)'`` and remove delay between steps (in all tours and modules)


```
2018-09-11 15:04:02,743 4675INFO openerp_test odoo.tests.common.phantomjs: Tour tour_pos_debt_notebook: step .o_pos_kanban button.oe_kanban_action_button succeeded
2018-09-11 15:04:05,777 4675INFO ? werkzeug: 127.0.0.1 - - [11/Sep/2018 15:04:05] "GET /web/static/src/img/spin.png HTTP/1.1" 200 -
2018-09-11 15:04:11,111 4675INFO openerp_test werkzeug: 127.0.0.1 - - [11/Sep/2018 15:04:11] "POST /web/dataset/call_button HTTP/1.1" 200 -
```

**Comment by ilmir-k** [comment]
@yelizariev please check the fix

**Comment by KolushovAlexandr** [comment]
@ilmir-k Too early I don't know the reason for this error

`page.evaluate eval expr: odoo.__DEBUG__.services['web_tour.tour'].tours.pos_printer_network_tour.ready
page.evaluate eval result: false`
it's not only for this module

**Comment by yelizariev** [comment]
@KolushovAlexandr check log below. Is it due to some unported code?

```
2018-09-14 15:20:32,385 4623 WARNING openerp_test odoo.tests.common.phantomjs: warning: Some modules could not be started
2018-09-14 15:20:32,385 4623 INFO openerp_test odoo.tests.common.phantomjs: Missing dependencies:     web.session,web.WebClient
2018-09-14 15:20:32,386 4623 INFO openerp_test odoo.tests.common.phantomjs: Non loaded modules:       web.AjaxService,web_editor.base,web_editor.ready,web_editor.ace,web.planner.common,web_tour.TourManager,web_tour.tour,web_editor.tour,web.web_client
```

File: pos_cashier_select/static/src/js/tour.js [hunk 2]
```diff
@@ -88,6 +88,7 @@ odoo.define('pos_cashier_select.tour', function(require) {
     steps = steps.concat({
         trigger: '.o_main_content:has(.loader:hidden)',
         content: 'waiting for loading to finish',
+        timeout: 11000,
```
Comment (yelizariev) [inline pos_cashier_select/static/src/js/tour.js]: make it bigger, please. We have around 10 sec for that step, so with 11 sec there is still big enough chance to get timeout error

File: pos_order_cancel_restaurant/static/src/js/tour.js [hunk 3]
```diff
@@ -11,12 +11,17 @@ odoo.define('pos_order_cancel_restaurant.tour', function(require) {
...
                     content: "<p>Click <b>table</b></p>",
-                    position: "bottom"
-                }
+                    position: "bottom",
+                    timeout: 15000,
+                }, {
+                    trigger: ".product-list .product",
+                    content: "<p>Click product 1</p>",
+                    position: "bottom",
+                },
```
Comment (yelizariev) [inline pos_order_cancel_restaurant/static/src/js/tour.js:24]: NOOOOOOOOOO! This comma breaks phantom tests!!!!!!!!!!!!!!!!

**Pull request merged**